Bottom: 15be28088889530c7ec38102f6d6b070ff4da9ad
Top:    6bba067412803fa6955b210331d18246e7cd5bf4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 18:01:23 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index b1bf5f0..b5a7cc7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -182,7 +182,7 @@ void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
